dolphin/db/sqlalchemy/api.py

```diff
@@ -148,6 +148,8 @@ def storage_get(context, storage_id):
     storage_by_id = this_session.query(Storage) \
         .filter(Storage.id == storage_id) \
         .first()
+    if storage_by_id is None:
+        raise exception.StorageNotFound(storage_id=storage_id)
```

Comment:
This does not seem correct to me. Usually any DB function should raise ORM Exceptions like ColumnError or NoResultFound etc..then this should be caught by the caller and raise the StorageNotFound error

Reply:
Here we are not looking for any DB exception. If query result is empty ( which is normal) we want to create an exception sayinf the queries storage_id is not in DB.

Reply:
ok. 
My only concern was that DB/Sqlalchemy should not raise any exception related to storage. It should be handled and raised by Storage APIs finally